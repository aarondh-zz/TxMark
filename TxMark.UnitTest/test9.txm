<div>
  (print: 2 * 5 - 7) = 3
  (print: 2 * 5 - 7 - 8 * 42) = -333
  (print: 2 * 5 - 7 - 55 * 3.141259265) = -169.783926
  (print: 2 * 5 * 1 - 7) = 3
  (print: 2 * 5 * 2 - 7) = 13
  (print: 2 * 5 - 7 * 1.5^2) = -5.75
  (print: 2 * (5 - 7) * 1.5^2) = -9
</div>
