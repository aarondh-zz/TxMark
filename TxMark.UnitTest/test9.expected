<div>
3 = 3
-340 = -333
-176.769259575 = -169.783926
3 = 3
13 = 13
-5.75 = -5.75
-9 = -9
</div>
